library/src/scala/tasty/Reflection.scala

```diff
@@ -2284,6 +2284,14 @@ class Reflection(private[scala] val internal: CompilerInterface) { self =>
     def caseFields(using ctx: Context): List[Symbol] =
       internal.Symbol_caseFields(sym)
 
+    /** Default parameters of a case class. The first elements of the pairs are names of
+     *  the parameters and values – symbols of the definitions sites of the default values.
+     *  Implementation restriction: only the default parameters in the first parameter group
+     *  are returned.
+     */
+    def defaultParams(using ctx: Context): List[(String, Symbol)] =
+      internal.Symbol_defaultParams(sym)
```

Comment:
Try to move the implementation here. There is no reason to add it to the `CompilerInterface` if it can be computed using it.